TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/bot/MovementServiceBinder.java

```diff
@@ -0,0 +1,58 @@
+package com.novoda.tpbot.bot;
+
+import android.content.ComponentName;
+import android.content.Context;
+import android.content.Intent;
+import android.content.ServiceConnection;
+import android.os.IBinder;
+
+class MovementServiceBinder {
```

Comment:
This code really handles the management of binding an activity to a `Service` 🤔  I wonder if we could make this more generic in future so that it takes a creator and **just** handles the state, the `connection` being present or not.